### ISSUE TITLE
how to make testset like lfw.bin, lfw_list.npy, data dir and so on

### ISSUE THREAD

**Issue by shiney5213** [peteryuX/arcface-tf2#6]
Hello,
I am studing arcface and I want to test my own dataset.
please how to make test dataset.
thank you for your help.

**Comment by HoangTienDuc** [comment]
hello @peteryuX how did you align your test dataset?

**Comment by peteryuX** [comment]
Hello @shiney5213 and @HoangTienDuc, nice to hear from you~
Right now, all the testing dataset are collected from https://github.com/ZhaoJ9014/face.evoLVe.PyTorch#Data-Zoo for providing a quick and robust evaluation.
But for your request, I will plan to provide the own testing dataset preparing when I am free.  Before then, you might need to dig into it by yourself. (the original data format: image data scale range are  [-1 ~ 1]. Paired images with equal and not-equal identity label.)

**Comment by aryachiranjeev** [comment]
> Hello @shiney5213 and @HoangTienDuc, nice to hear from you~ Right now, all the testing dataset are collected from https://github.com/ZhaoJ9014/face.evoLVe.PyTorch#Data-Zoo for providing a quick and robust evaluation. But for your request, I will plan to provide the own testing dataset preparing when I am free. Before then, you might need to dig into it by yourself. (the original data format: image data scale range are [-1 ~ 1]. Paired images with equal and not-equal identity label.)

Hi @peteryuX, now if you have done this, how to make our own custom test data preparing file, can you please share it? 
Or can you please tell how the pairs are created in lfw due to which lfw_list contains list of True and False.
(As lfw_list.npy has 6,000 elements whereas lfw dataset has 12,000 images of size(3,112,112)) :- it's confusing can you please help in this?
Thanks

**Comment by BurakaKrishna** [comment]
@aryachiranjeev For lfw_list explanation refer to https://github.com/leondgarse/Keras_insightface and refer to https://github.com/deepinsight/insightface/issues/791 for creating your own dataset
